iMichaelOwolabi/my-diarium-backend
# 167951199 Setup graphql server for the app

#### What does this PR do?
Setup graphql server for the app
#### Description of Task to be completed?
Setup graphql server for the app
#### How should this be manually tested?
Clone the project repo and cd into the directory
- run npm  install to install all dependencies
- run npm run dev
- You should see the project server running
- visit http://localhost:5000/graphql
- you should see the graphql playground
#### Any background context you want to provide?
This task is just to setup graphql server for the app
#### What are the relevant pivotal tracker stories?
[#167951199](https://www.pivotaltracker.com/story/show/167951199)
